ko/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: news_post
+title: "루비 1.9.3 지원 종료"
+author: "Olivier Lacan"
+translator: "marocchino"
+date: 2015-02-23 00:00:00 +0000
+lang: ko
+---
+
+오늘(2015년 2월 23일)부터 루비 1.9.3의 모든 지원이 종료됩니다. 루비 최신 버전에
+적용하는 버그 수정, 보안 패치는 더는 1.9.3에 백 포트되지 않습니다.
+
+이 지원 종료는 [1년 전부터 공지](https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/)했었습니다.
```

Comment:
Korean translation is available: https://www.ruby-lang.org/ko/news/2014/01/10/ruby-1-9-3-will-end-on-2015/
:wink: